### DISCUSSION TITLE
Payment Intent is not accepting payment method

### DISCUSSION THREAD

**Discussion by ashishdock** [flutter-stripe/flutter_stripe#1237] [Q&A]
**Describe the bug**
I have been trying to get it to work for 6 hours,but it keeps giving me "You cannot confirm this PaymentIntent because it's missing a payment method. You can either update the PaymentIntent with a payment method and then confirm it again, or confirm it again directly with a payment method.","payment_intent_unexpected_state",.
I am entering the card 4242... as per documentation.

**To Reproduce**
1. Installed latest version of Flutter_Stripe
2. Followed the complete documentation
3. entered 4242... card number with future date for expiry and any5 digit number for zip
4. The payment sheet disappears without any acknowledgement about the payment.
5. When confirming the status of the client_intent id, it shows that the payment method is missing.

**Expected behavior**
How to get the correct card payment to get accepted. Why is this error not elaborate as to what the the developer needs to do to fix it?

**Smartphone / tablet**
 - Android emulator
 - OS: Android
- everything is configured as per your documentation

**Comment by Anonymousgaurav** [comment]
anyone got solution?
